windows/README.md

```diff
@@ -111,3 +111,15 @@ Set-Item WSMan:localhost\client\trustedhosts -value *
 You can confirm this ran successfully with, `get-item WSMan:\localhost\Client\TrustedHosts`, which should list an output with a `*` under the Value column.
 
 After running this command you should be able to try again and successfully connect to the remote hosts to gather data.
+
+### Server name cannot be resolved error
+
+Run ping \<subject-ipv4\>. If there's no response, you need to take steps to ensure these servers can communicate with each other.
+
+Ensure the security group of the subject VMs allows incoming connections from the controller machine.
```

Comment:
This line and the "after-next one" (line 123) read very similarly.  maybe we can remove this one. Also, this line mentions security groups that I don't think might be applicable to our audience/clients.